plugins/ad4m/skills/ad4m/SKILL.md

```
@@ -217,25 +229,27 @@ The waker now tells you which **channel** the event happened in, so you know exa
 
 ### First-time setup
 
+In **managed mode** (default), the plugin auto-manages everything. Just install the plugin:
+
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)